core/src/main/java/com/topjohnwu/superuser/Shell.java

```diff
@@ -77,15 +77,15 @@ public abstract class Shell implements Closeable {
      */
     public static final int ROOT_SHELL = 1;
     /**
-     * Shell status: Root shell with mount master enabled.
-     * One possible result of {@link #getStatus()}.
+     * @deprecated Not used anymore.
      * <p>
      * Constant value {@value}.
      */
+    @Deprecated
     public static final int ROOT_MOUNT_MASTER = 2;
```

Comment:
```suggestion
```

Reply:
The value should be preserved to prevent code breakage